src/components/hurricane-marker.scss

```diff
@@ -47,6 +55,12 @@
     display: flex;
     align-items: center;
     justify-content: center;
+    // Text outline
+    text-shadow: 1px 1px 0 #fff,
+      -1px -1px 0 #fff,
+      1px -1px 0 #fff,
+      -1px 1px 0 #fff,
+      1px 1px 0 #fff;
```

Comment:
There will be a conflict here since I added something similar for the track category markers. I removed the last entry, however, as it seems redundant.